### PR TITLE
Add validation losses

### PR DESCRIPTION
**Pull request by sushen25**

### CODE REVIEW

**Review by voirra — APPROVED** [review]

**Review by voirra — APPROVED** [review]

File: alt_train.py
Comment (sushen25) [inline alt_train.py]: Just a question,
so ur storing both the training and validation losses in the same .npy file?

Comment (sushen25) [reply alt_train.py]: nvm figured it out, it's 2 separate files dw

Comment (voirra) [reply alt_train.py]: actually I found something I missed, hold off from merging

**Comment by voirra** [comment]
rebased onto master, will merge this after I'm happy that the validation losses works

**Pull request merged**
